sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+### Tofino
+
+_Tofino_ takes up Welsby's spatialization of time and his wind-camera coupling to create a somewhat different kind of system. It consists of a series of long takes of breaking surf captured at Wikaninnish Beach near Tofino, British Columbia. The first part of the video is structured as a "temporal panorama": each static shot moves the camera a frame or so to the right, cumulatively representing a 180° view of the beach<sup>16</sup>. Space, in a sense, is temporalized by the way the shots are sequenced. Time, too, is spatialized in _Tofino_. Two versions of each shot are composited or layered together: one layer runs at the standard 24 frames per second (cinema's "real" time), while the other layer gently speeds up and slows down in an oscillating rhythm. A simple Flocking signal generator, which outputs a slowly-modulated triangle wave, is connected to the playback rate of the latter video layer, causing it to move faster and slower in a motion that echoes the swell and break of the waves on the beach. Composited together, we see the two timelines moving in and out of phase. When the modulated layer runs more slowly than realtime, we see breaking waves echoed moments later by their composited ghosts. When it moves faster, actions are anticipated ahead of "real" time: we see a future.
+
+This technical-elemental coupling uniquely makes visible, in a way that might not be possible without such an a video apparatus, the otherwise invisible mutability and bidirectional flux of time. Or perhaps this system actually brings into being--makes real--a new kind of time via its mediations and modulations?
```

Comment:
Comment from David: "I like this suggestion, but it could be fleshed out (especially in light of the Epstein ’new form of thinking’ citation): 'Or perhaps this system actually brings into being--makes real--a new kind of time via its mediations and modulations?' (I love the description of the piece."